config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Doorkeeper.configure do
+  orm :active_record
+
```

Comment:
```suggestion
  api_only

```

According to the documentation there is an [api_only](https://doorkeeper.gitbook.io/guides/ruby-on-rails/api-mode) mode. 🤔 Did you try it? :)

> Note that I never worked with DoorKeeper, so this suggestion might not work 😇

Reply:
I tried adding this flag but `rails routes` still returns `applications` routes so I still have to skip them. 🙏

Reply:
While routes might still be present, I understand from the documentation that using the `api_only` flag can reduce the features loaded into the project (like views, view helpers, etc...). So even if it feels like it does not change anything, it might still be useful for the project :)